ansible_collections/arista/avd/docs/plugins/Modules and action plugins/eos_validate_state_reports.md

```diff
@@ -0,0 +1,65 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# eos_validate_state_reports
+
+!!! note
+    Always use the FQCN (Fully Qualified Collection Name) `arista.avd.eos_validate_state_reports` when using this plugin.
+
+!!! warning "This module is in **preview** mode"
+    This module is not guaranteed to have a backwards compatible interface.
+
+Generates validation reports for the eos\_validate\_state role
+
+## Synopsis
+
+The <code>arista.avd.eos\_validate\_state\_reports</code> module, an Ansible Action Plugin, is designed to generate validation reports from the test results produced by the ANTA test framework.
+
+This plugin requires a temporary JSON file for each host in the Ansible play, which contains all test results. The path to this temporary JSON file, used by the plugin, is supplied in the hostvars by the \`eos\_validate\_state\_runner\` plugin.
```

Comment:
Likewise here, the syntax highlighting is not working on the preview site with these escape characters

Reply:
This file is generated by our tooling. Some of the over-eager escaping is an artifact of that.